mathics/builtin/graphics.py

```diff
@@ -1268,14 +1284,24 @@ def _get_image_size(self, options, graphics_options, max_width):
                 aspect = None
 
         image_size = graphics_options['System`ImageSize']
-        image_size = image_size.get_name()
-        base_width, base_height = {
-            'System`Automatic': (400, 350),
-            'System`Tiny': (100, 100),
-            'System`Small': (200, 200),
-            'System`Medium': (400, 350),
-            'System`Large': (600, 500),
-        }.get(image_size, (None, None))
+        try:
+            if image_size.is_numeric():
+                base_width = int(image_size.to_number())
```

Comment:
Possible minor bug here: If image_size is `Symbol(Pi)` for example. I'd say it's not worrying about non-integer inputs so checking `isinstnace(image_size, Integer)` on the line above and then using `get_int_value()` would be fine.